content/influxdb/clustered/install/customize-cluster/scale.md

```diff
@@ -0,0 +1,303 @@
+---
+title: Customize your cluster scale
+seotitle: Customize the scale of your InfluxDB cluster
+description: >
+  Customize the scale of your cluster to best suit your workload.
+menu:
+  influxdb_clustered:
+    name: Customize cluster scale
+    parent: Customize your cluster
+weight: 201
+related:
+  - /influxdb/clustered/admin/scale-cluster/
+---
+
+InfluxDB Clustered lets you scale each component in your cluster individually,
+so you can customize your cluster's scale to address the specific the specific
+needs of your workload.
+For example, if you have a heavy write workload, but not a heavy query workload,
+you can scale your Router and Ingester both [vertically](/influxdb/clustered/admin/scale-cluster/#vertical-scaling)
+and [horizontally](/influxdb/clustered/admin/scale-cluster/#horizontal-scaling)
+to increase your write throughput and latency.
+
+## Default scale settings
+
+- **1 router**:
+  Additional routers increase your cluster's ability to handle concurrent write
+  requests.
+- **3 ingesters**:
+  Ensures redundancy on the write path.
+- **1 compactor**:
+  While you can have multiple compactors, it is more efficient to scale the
+  compactor vertically (assign more CPU and memory) rather than horizontally
+  (increase the number of compactors).
+- **1 querier**:
+  The optimal number of queriers depends on the number of concurrent queries you
+  are likely to have and how long they take to execute.
+
+The default values provide a good starting point for testing.
+Once you have your cluster up and running and are looking for scaling
+recommendations for your anticipated workload, please
+[contact the InfluxData Support team](https://support.influxdata.com).
+We are happy to work with you to identify appropriate scale settings based on
+your anticipated workload.
+
+## Customize scale settings
+
+Your `AppInstance` resource controls the scale of components in your InfluxDB
+cluster. You can edit `AppInstance` resource directly or, if using the
+[InfluxDB Clustered Helm chart](https://github.com/influxdata/helm-charts/tree/master/charts/influxdb3-clustered)
+to manage your deployment, you can edit resource settings in your `values.yaml`.
+
+{{% note %}}
+_For specific scaling recommendations and guidelines, see
+[Scale your cluster](/influxdb/clustered/admin/scale-cluster/)._
+{{% /note %}}
+
+With Kubernetes, you can define both the minimum resources to request for each
+component and resource limits to apply to each component.
+
+{{< tabs-wrapper >}}
+{{% tabs "medium" %}}
+[AppInstance](#)
+[Helm](#)
+{{% /tabs %}}
+
+{{% tab-content %}}
+
+<!----------------------------- BEGIN APPINSTANCE ----------------------------->
+
+**To use custom scale settings for your InfluxDB cluster**, edit values for the
+following fields in your `myinfluxdb.yml`.
+If omitted, your cluster uses the default scale settings.
+
+- `spec.package.spec.resources`
```

Comment:
I realize you're mostly just reorganizing existing documents, but it might be worth mentioning the `garbage-collector` as a component with scalable resources here.

Reply:
Thanks @waynr. I'll add it. The GC isn't included in the [`example-customer.yml`](https://github.com/influxdata/k8s-idpe/blob/a98ab88f08f2ea911960ab2384b14cac5f15913a/env/clustered/example-customer.yml#L255-L289) which is why I didn't include it here.

- Is the key `garbage-collector` or `garbage_collector`?
- I know at one point it did honor the `replicas` option, but it breaks when adding replicas. I'll keep this one out of the examples, but did we remove support or limit multiple replicas for the GC?